src/app/shell/result/workshop-map-view-list/workshop-map-view-list.component.ts

```diff
@@ -43,8 +43,10 @@ export class WorkshopMapViewListComponent implements OnInit {
   widthOfWorkshopCard = Constants.WIDTH_OF_WORKSHOP_CARD;
 
   ngOnInit() {
-    this.workshops = this.filteredWorkshops?.entities;
-  
+    this.filteredWorkshops$.subscribe(filteredWorkshops => {
```

Comment:
use type;

seems it may cause performance issue, unsubscribing is required here